pages/[id]/bridge.tsx

```diff
@@ -310,10 +311,14 @@ const Bridge = () => {
                   {mounted ? formatUnits(source.fee, 18) : 0} ETH
                 </div>
               </div>
-              <div className="text-center">
-                {mounted &&
-                /* ⚪️ First, if we are on the wrong network, prompt to switch networks. Add new network to user wallet if needed. */
-                source.chain.id !== chain?.id ? (
+              <div className="flex text-center justify-center">
+                {mounted && !address ? (
+                  <div className="mt-5">
+                    <ConnectButton />
+                  </div>
+                ) : mounted &&
+                  /* ⚪️ First, if we are on the wrong network, prompt to switch networks. Add new network to user wallet if needed. */
+                  source.chain.id !== chain?.id ? (
```

Comment:
Everything is working great. It may be better if both of these chained turnaries are abstracted outside of the JSX in order to improve readability.